definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-alt-service-process-server-nonprod.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "LiveFrom": "12/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "serveByProcessServer",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "CRU"
```

Comment:
I think you're right to have CRU here and the story is wrong, but can you confirm with Jeremy?

Reply:
Waiting on response

Reply:
Why do we want "CRU" on this one? The courtadmin user is not really used from our side right?

Reply:
The AC states "A Caseworker can trigger event 'serveByProcessServer' from 'GeneralConsiderationComplete'. This moves case into the state 'AwaitingProcessServerService'."
As we have two roles for Caseworker, i thought it was appropriate to have this

Reply:
Confirmed by Jeremy that this should be R